scripts/x-data-grid.exports.json

```diff
@@ -287,11 +287,12 @@
   { "name": "GridFilterInputDate", "kind": "Function" },
   { "name": "GridFilterInputDateProps", "kind": "TypeAlias" },
   { "name": "GridFilterInputMultipleSingleSelect", "kind": "Function" },
-  { "name": "GridFilterInputMultipleSingleSelectProps", "kind": "Interface" },
+  { "name": "GridFilterInputMultipleSingleSelectProps", "kind": "TypeAlias" },
   { "name": "GridFilterInputMultipleValue", "kind": "Function" },
   { "name": "GridFilterInputMultipleValueProps", "kind": "TypeAlias" },
   { "name": "GridFilterInputSingleSelect", "kind": "Function" },
   { "name": "GridFilterInputSingleSelectProps", "kind": "TypeAlias" },
+  { "name": "GridFilterInputSlotProps", "kind": "TypeAlias" },
```

Comment:
Is this export necessary?

Reply:
I think so, custom filters with an input component would base their typings on that.